clang/lib/Driver/Driver.cpp

```diff
@@ -400,6 +400,13 @@ DerivedArgList *Driver::TranslateInputArgs(const InputArgList &Args) const {
       continue;
     }
 
+    if (A->getOption().matches(options::OPT_offload_lib_Group)) {
+      if (!A->getNumValues()) {
+        Diag(clang::diag::warn_drv_unused_argument) << A->getSpelling();
+        continue;
+      }
```

Comment:
Could you add a test for this?

Reply:
Hi, already add a test for this commit.
Thanks very much.